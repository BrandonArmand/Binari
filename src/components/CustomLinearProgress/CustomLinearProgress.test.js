import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import CustomLinearProgress from "./CustomLinearProgress";

describe("CustomLinearProgress", () => {
    it("renders snapshot", () => {
        const { container } = render(<CustomLinearProgress />);
        expect(container).toMatchSnapshot();
    });
    it("renders CustomLinearProgress", () => {
        render(<CustomLinearProgress />);
        expect(screen.getByRole("progressbar")).toBeInTheDocument();
    });
    it("renders CustomLinearProgress", () => {
        render(<CustomLinearProgress color="primary" />);
        expect(screen.getByRole("progressbar")).toBeInTheDocument();
    });
});
