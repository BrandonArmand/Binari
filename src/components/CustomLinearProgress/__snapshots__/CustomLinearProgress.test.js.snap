// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomLinearProgress renders snapshot 1`] = `
<div>
  <div
    class="MuiLinearProgress-root makeStyles-root-1 makeStyles-grayBackground-16 MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
    role="progressbar"
  >
    <div
      class="MuiLinearProgress-bar makeStyles-bar-2 makeStyles-gray-9 MuiLinearProgress-bar1Indeterminate MuiLinearProgress-barColorPrimary"
    />
    <div
      class="MuiLinearProgress-bar makeStyles-bar-2 makeStyles-gray-9 MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
    />
  </div>
</div>
`;
